### PR TITLE
16 ev3dev

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

File: news/_posts/2016-10-15-kernel-release-cycle-16.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 16"
+---
+
+Nothing too exciting in this release. Most of the changes are for adding support
+for a new BeagleBone cape that is being developed.
+
+Also support for polling the `status` attribute of the tacho motor class has
```
Comment (WasabiFan) [inline news/_posts/2016-10-15-kernel-release-cycle-16.md]: Could you change this opening sentence of this paragraph to make it flow better? Something like "This version adds support for the..."

File: news/_posts/2016-10-15-kernel-release-cycle-16.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 16"
+---
+
+Nothing too exciting in this release. Most of the changes are for adding support
```
Comment (WasabiFan) [inline news/_posts/2016-10-15-kernel-release-cycle-16.md]: Use a complete sentence; prepend "There's".

**Comment by dlech** [comment]
Updated. I'm thinking about adding a code sample for polling the motor attribute as well.

**Comment by WasabiFan** [comment]
Ah, good idea! A Python example would probably be a great demo which the majority of users could understand. The polling feature might be small, but it can make some things a lot easier!

File: news/_posts/2016-10-15-kernel-release-cycle-16.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 16"
+---
+
+There is nothing too exciting in this release. Most of the changes are for
+adding support for a new BeagleBone cape that is being developed.
+
+There is one bug fix. Support for polling the `status` attribute of the tacho
```
Comment (dlech) [inline news/_posts/2016-10-15-kernel-release-cycle-16.md]: doh...

**Comment by dlech** [comment]
updated with python example

**Comment by ddemidov** [comment]
Cool! This could be wrapped nicely into the ev3dev-lang-python.

**Comment by WasabiFan** [comment]
By the way, you can have GitHub rebase your commits onto master; it's one of the merge types.

**Pull request merged**